automatic/ketarin/Readme.md

```diff
@@ -0,0 +1,3 @@
+# [<img src="https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/2963b7e22c35b0dfb637bb1d65ab74eb298bc66c/icons/ketarin.png" height="48" width="48" /> ketarin](https://chocolatey.org/packages/ketarin)
```

Comment:
Add Features from the site.

Reply:
ah, I missed the entire feature section on the website. I'll get that updated.

Reply:
updated